bee-api/bee-rest-api/src/endpoints/routes/api/v1/info.rs

```diff
@@ -85,9 +85,9 @@ pub(crate) async fn info<B: StorageBackend>(
             bech32_hrp,
             min_pow_score: protocol_config.minimum_pow_score(),
             rent_structure: RentStructureResponse {
-                vbyte_cost: 500,      // TODO
-                vbyte_factor_data: 1, // TODO
-                vbyte_factor_key: 10, // TODO
+                vbyte_cost: protocol_config.byte_cost().byte_cost,
+                vbyte_factor_data: protocol_config.byte_cost().weight_for_data,
+                vbyte_factor_key: protocol_config.byte_cost().weight_for_key,
```

Comment:
A suggestion: maybe we should use the names provided by the TIP for consistency ?

Reply:
So just for clarification: You propose changing the members of `ByteCostConfig` to `vbyte_cost`, `vbyte_factor_data`, and `vbyte_factor_key`?

Reply:
If these are the actual names used in the TIPs then yes, I usually try to mimic the TIPS as much as possible. But your call really.